flink-formats/flink-avro/src/test/java/org/apache/flink/table/runtime/batch/AvroTypesITCase.java

```diff
@@ -240,7 +240,7 @@ public void testAvroToAvro() {
     }
 
     private DataStream<User> testData(StreamExecutionEnvironment env) {
-        return env.fromElements(USER_1, USER_2, USER_3);
+        return env.fromElements(USER_1, USER_2, USER_3).setParallelism(1);
```

Comment:
huh. These changes does make one wonder whether we shouldn't set the parallelism to 1 by default, but allow the user to override it 🤔

Reply:
✅ https://github.com/apache/flink/pull/23553/commits/c04d0dd540c6c0e1635fcf59eb3c10a46cd1e0e5